bin/MadGraph5_aMCatNLO/cards/production/13TeV/tH_t_topdecay_LH_narrow_M1100/tH_t_topdecay_LH_narrow_M1100_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.1                 2014-09-25         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model STP_UFO_freeWidth
+define Topo = t23 t23~
+define bb = b b~
+define WW = w+ w-
+define tt = t t~
+define ferm =  ve vm vt ve~ vm~ vt~ u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+
+generate p p > j tt Topo  / h ,  tt > bb WW , ( Topo > h tt / z w+ w- ,  tt > bb WW )
```

Comment:
In Run1, samples were genertd only up to the T. All subsequent decays were done in Pythia6. If the width treatment between Pythia6 and Pythia8 are the same, we will get the same result. 
In any case this will be negliglible for MT in case of lv decay and M in case of jj decay. So a narrow-width approximation does not compromise the results.